openshift/templates/fider/fider-bcgov.dc.yaml

```diff
@@ -0,0 +1,296 @@
+---
+kind: Template
+apiVersion: v1
+message: A new Fider application, customized with noTLS, has been created in your project
+metadata:
+  name: nrmf-feedback-dc
+parameters:
+- description: The OpenShift Namespace where the ImageStream resides.
+  displayName: Namespace
+  name: IS_NAMESPACE
+  value: b7cg3n-tools
+- description: The OpenShift ImageStream name.
+  displayName: Image Stream
+  name: IS_NAME
+  value: fider-bcgov
+- description: Version of Feedback image to be used (e.g. v0.18.0)
+  displayName: Version of Fider Product Feedback
+  name: IS_VERSION
+  value: 0.18.0
+- description: The short-name of the Feedback Product, lower-case with hyphen.
+  displayName: Feedback Product Name
+  name: FEEDBACK_NAME
+  required: true
+  value: nrm
+- description: The Fider environment (development | production).
+  displayName: Fider Go Environment
+  name: GO_ENV
+  value: production
+- description: The Fider loggin level (DEBUG | INFO).
+  displayName: Fider application logging level
+  name: LOG_LEVEL
+  value: INFO
+- description: JWT Secret.
+  displayName: Fider Go Environment
+  from: "[a-zA-Z0-9]{32}"
+  generate: expression
+  name: JWT_SECRET
+  required: true
+- description: Email SMTP Host Configuration
+  displayName: SMTP Host
+  name: EMAIL_SMTP_HOST
+  value: apps.smtp.gov.bc.ca
+- description: Email SMTP Port Configuration
+  displayName: SMTP Port
+  name: EMAIL_SMTP_PORT
+  value: '25'
+- description: Email SMTP Username
+  displayName: SMTP User
+  name: EMAIL_SMTP_USERNAME
+  value: Gary.T.Wong@gov.bc.ca
```

Comment:
is Gary's email the correct value for our app? I can't fully tell what this is being used for

Reply:
It's not, it just has to be set correctly when applying the template thought, so no need to change it.